### DISCUSSION TITLE
Nada lang assisstant GPT

### DISCUSSION THREAD

**Discussion by DeMonkeyCoder** [NillionNetwork/discussions#41] [Show and tell]
CustomGPTs are helpful for onboarding developers. I created one and integrated Nada lang's docs and examples as its knowledge base. I think it can be helpful with a more detailed documentation as its knowledge base
https://chat.openai.com/g/g-J8KG9NCSy-nada-dev-helper 

![image](https://github.com/NillionNetwork/discussions/assets/34129327/69c6d052-96a1-432f-94ac-529dbe493cbe)

**Comment by wwwehr** [comment]
This is so good
